src/index.tsx

```diff
@@ -6,8 +7,10 @@ import store from './app/store'
 import './app/styles.scss'
 
 ReactDOM.render(
+  <MuiThemeProvider>
```

Comment:
We need to remove this if we are removing material ui

Reply:
👍